### DISCUSSION TITLE
What evaluation means in MONAI?

### DISCUSSION THREAD

**Discussion by OmarAshkar** [Project-MONAI/MONAI#6982] [Q&A]
Hello,

Thank you for maintaining MONAI. I want to ask what does evaluation means in MONAI. `SupervisedEvaluator`  will take the validation set, but does the training being optimized also by that set or not? It is frequent to Train/val/test split in MONAI apps, but I never figured if the validation and test should be different or not.

Thank you!

**Comment by KumoLiu** [comment]
Hi @OmarAshkar, thanks for your interest here.
`SupervisedEvaluator` can do evaluation or inference and it depends on what data you feed to it. To ensure that no data from the test set is leaked out, we typically divide the data into training, validation, and testing.

I hope I explained it clearly. Thanks!
